yarn.lock

```diff
@@ -21,7 +26,7 @@ ajv@^4.9.1:
     co "^4.6.0"
     json-stable-stringify "^1.0.1"
 
-algoliasearch-helper@0.0.0-5a0352a, algoliasearch-helper@2.26.1, algoliasearch-helper@^2.26.0:
+algoliasearch-helper@2.26.1, algoliasearch-helper@^2.26.0, algoliasearch-helper@^3.2.2:
   version "2.26.1"
```

Comment:
interesting, this seems wrong, the helper version needs to be v3 for InstantSearch v4